torchvision/models/detection/rpn.py

```diff
@@ -27,6 +27,56 @@ def _onnx_get_num_anchors_and_pre_nms_top_n(ob, orig_pre_nms_top_n):
 
     return num_anchors, pre_nms_top_n
 
+@torch.jit.unused
+def _onnx_get_grid_image_sizes(feature_maps, image_list_tensors):
+    # type: (List[Tensor], Tensor) -> Tuple(List[Tensor], Tensor)
```

Comment:
I believe there is a typo here:
```python
Typle(List[Tensor], Tensor)
```
should probably read
```python
Tuple[List[Tensor], Tensor]
```

Reply:
Sorry, I may not have time to tidy my tests recently.  I'll do this if I have free cycles to spin.

Reply:
Ok, no worries, thanks for the initial PR!

@lara-hdr do you think someone in your team could help get this PR have a green build, including tests to make sure that RPN now supports arbitrary input shapes?

Reply:
Yes, some tests are failing. I’ll do this first thing after the weekend.